operator/v1alpha1/types_image_content_source_policy.go

```diff
@@ -0,0 +1,50 @@
+package v1alpha1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ImageContentSourcePolicy holds cluster-wide information about how to handle registry mirror rules.
+// The canonical name is `cluster`
+type ImageContentSourcePolicy struct {
+	metav1.TypeMeta `json:",inline"`
+	// Standard object's metadata.
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// spec holds user settable values for configuration
+	// +required
+	Spec ImageContentSourcePolicySpec `json:"spec"`
+}
+
+// ImageContentSourcePolicySpec is the specification of the ImageContentSourcePolicy CRD.
+type ImageContentSourcePolicySpec struct {
+	// repositoryDigestMirrors allows images referneced by image digests in pods to be
+	// pulled from alternative mirrored repository locations. The image pull specification
+	// provided to the pod will be compare to the source locations described in RepositoryDigestMirrors
+	// and the image may be pulled down from any of the repositories in the list instead of the
+	// specified repository allowing administartors to choose a potentially faster mirror.
```

Comment:
typo: `administartors`

Reply:
We could also mention resiliency as an advantage/use case for this feature.

Reply:
fixed